RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -23,7 +23,11 @@
         'keep *_inclusiveCandidateSecondaryVerticesCvsL_*_*',
         'keep *_pfInclusiveSecondaryVertexFinderCvsLTagInfos_*_*',
         'keep *_pfCombinedCvsLJetTags_*_*',
-        'keep *_pfCombinedCvsBJetTags_*_*'
+        'keep *_pfCombinedCvsBJetTags_*_*',
+        # DeepFlavour -- wait for re-reco to include it
+        # 'keep *_pfDeepCSVJetTags_*_*',
+        # DeepCMVA -- wait for re-reco to include it
+        # 'keep *_pfDeepCMVAJetTags_*_*',
```

Comment:
rereco is only for data. 
This feature is relevant for both data and MC.
AOD data and MC event content will have to remain consistent.
So, these taggers will need to be configured as 
- additions that can be added on top of AOD on the fly for those who have to use AOD [via a recipe]
- as an extra b-tagger info embedded in miniAOD jets

Other data/MC asymmetric alternatives are technically possible, but do not appear the most practical.

The comment in the code should probably be edited or removed completely, to be replaced by a customization method to enable it on top of AOD (unless you want the recipe to be "edit these lines").